diff --git a/lua/neotest-vstest/vstest/client.lua b/lua/neotest-vstest/vstest/client.lua
index cf1c64f..20e2155 100644
--- a/lua/neotest-vstest/vstest/client.lua
+++ b/lua/neotest-vstest/vstest/client.lua
@@ -164,7 +164,7 @@ function M.debug_tests(runner, ids)
   local max_wait = 30 * 1000 -- 30 sec
 
   cli_wrapper.spin_lock_wait_file(pid_path, max_wait)
-  local pid = lib.files.read(pid_path)
+  local pid = nio.fn.trim(lib.files.read(pid_path))
   return pid, on_attach, process_output_path, result_stream_path, result_path
 end
 
